setup.py

```diff
@@ -188,6 +188,10 @@ def get_winver():
 
 elif FREEBSD:
     macros.append(("PSUTIL_FREEBSD", 1))
+    libraries = ["devstat"]
+    os_major = int(platform.release().split(".")[0])
+    if os_major >= 8:
+        libraries.append("kvm")
```

Comment:
Is this really necessary? It wasn't thus far. I'm worried about the string parsing. If it fails, the whole installation breaks.

Reply:
Thanks for the reminder.
As mentioned, I actually more or less rebased the changes I made years ago, and for whatever reason it seemed neccessary then - probably I used a `.0` FreeBSD version which had some library glitches.

With all the (FreeBSD) versions mentioned in the commit log, it is not neccessary to specific `libkvm` at all, as `libdevstat` pulls it already!